appknox/client.py

```diff
@@ -518,11 +537,12 @@ def rescan(self, file_id: int) -> int:
 class ApiResource(object):
     def __init__(
         self, host: str = DEFAULT_API_HOST, headers: object = None,
-        auth: Dict[str, str] = None
+        auth: Dict[str, str] = None, proxies: object = None
     ):
         self.host = host
         self.headers = {**headers}
         self.auth = auth
+        self.proxies = proxies
```

Comment:
Would it be better to do something like this?
`if proxies:
  self.session = requests.Session()
  self.session. proxies.update(proxies)`

Reply:
done